models/app.model.js

```diff
@@ -34,3 +34,14 @@ exports.readArticles = () => {
     return rows;
   });
 };
+
+exports.readCommentsByArticleId = (id) => {
+  return db
+    .query("SELECT * FROM comments WHERE article_id = $1", [id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
```

Comment:
If an article exists but has no comments should that return an error? thinking how we interact with social media for instance, if you check the comments on a post but their are none, you wouldnt expect an error, you would expect it just to be empty.